.pre-commit-config.yaml

```diff
@@ -1,51 +1,19 @@
 default_language_version:
-    python: python3.11 # Using .python-version would be nice
+    python: python3.12 # Using .python-version would be nice
```

Comment:
Why specify `3.12` specifically? I always just do `python3`

Reply:
I vaguely recall encountering something that needed a specific version, and I've been specific ever since. Making everything `system` removes the need (and means just uv and yarn need caching).